src/components/movie-viewer/MovieDetails.jsx

```diff
@@ -16,15 +18,51 @@ export default function MovieDetails() {
 
   useEffect(() => {
     getData();
-    console.log(movie);
-  }, []);
+  }, []); // eslint-disable-line
```

Comment:
I think you can remove this dependency instead of adding an exception to the eslint. Unless that causes a problem.